### PR TITLE
Add DD_TRACE_HTTP_POST_DATA_PARAM_ALLOWED's doc

### PR DESCRIPTION
**Pull request by PROFeNoM**
### What does this PR do?
This PR aims to documented a newly introduced option that was added as part of [#1971](https://github.com/DataDog/dd-trace-php/pull/1971) in dd-trace-php.
---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Check images for PII
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

File: content/en/tracing/trace_collection/library_config/php.md [hunk 1]
```diff
@@ -313,6 +313,20 @@ Set an application's version in traces and logs, for example: `1.2.3`, `6c44da20
 **Default**: `*`<br>
 A comma-separated list of query parameters to be collected as part of the URL. Set to empty to prevent collecting any parameters, or `*` to collect all parameters. Added in version `0.74.0`.
 
+`DD_TRACE_HTTP_POST_DATA_PARAM_ALLOWED`
+: **INI**: `datadog.trace.http_post_data_param_allowed`<br>
+**Default**: ""<br>
+A comma-separated list of HTTP POST data fields to be collected as part the request. Set to empty by default to redact all posted values, or `*` to collect and redact them if needed. When specific fields are explicitly whitelisted, only those fields' values aren't redacted. Added in version `0.86.0`.<br>
```
Comment (buraizu) [inline content/en/tracing/trace_collection/library_config/php.md:319]:
```suggestion
A comma-separated list of HTTP POST data fields to be collected as part the request. Leave empty to redact all posted values, or use `*` to collect and redact them if needed. When specific fields are explicitly allowed, only those fields' values remain. Added in version `0.86.0`.<br>
```

File: content/en/tracing/trace_collection/library_config/php.md [hunk 2]
```diff
@@ -313,6 +313,20 @@ Set an application's version in traces and logs, for example: `1.2.3`, `6c44da20
...
+**Example**: 
+  - The posted data is `foo[baz]=bar&foo[baz][bar]=quz&foo[bar][password]=baz&foo[bar][baz]=quz`
+  - `DD_TRACE_HTTP_POST_DATA_PARAM_ALLOWED` is set to `foo.baz,foo.bar.password`<br>
+  - In this scenario, the collected metadata is:
+    - `http.request.foo.baz=bar`
+    - `http.request.foo.baz.bar=quz`
+    - `http.request.foo.bar.password=baz`
+    - `http.request.foo.bar.baz=<redacted>`
+
+
```
Comment (buraizu) [inline content/en/tracing/trace_collection/library_config/php.md:329]:
```suggestion
```

**Review by buraizu — CHANGES_REQUESTED** [review]
Thanks for the PR, just left some suggestion per our [style guide](https://github.com/DataDog/documentation/blob/master/CONTRIBUTING.md).

File: content/en/tracing/trace_collection/library_config/php.md [hunk 3]
```diff
@@ -316,7 +316,7 @@ A comma-separated list of query parameters to be collected as part of the URL. S
 `DD_TRACE_HTTP_POST_DATA_PARAM_ALLOWED`
 : **INI**: `datadog.trace.http_post_data_param_allowed`<br>
 **Default**: ""<br>
-A comma-separated list of HTTP POST data fields to be collected as part the request. Set to empty by default to redact all posted values, or `*` to collect and redact them if needed. When specific fields are explicitly whitelisted, only those fields' values aren't redacted. Added in version `0.86.0`.<br>
+A comma-separated list of HTTP POST data fields to be collected as part the request. Leave empty to redact all posted values, or use `*` to collect and redact them if needed. When specific fields are explicitly allowed, only those fields' values remain. Added in version `0.86.0`.<br>
```
Comment (buraizu) [inline content/en/tracing/trace_collection/library_config/php.md:319]:
```suggestion
A comma-separated list of HTTP POST data fields to be collected as part the request. Leave empty to redact all posted values, or use `*` to collect them all and redact if needed. When specific fields are explicitly allowed, only those fields' values remain. Added in version `0.86.0`.<br>
```

**Review by buraizu — APPROVED** [review]
Thanks for implementing those changes. I'm approving with one last optional suggestion for clarity. I might not be understanding the expected behavior here, so feel free to ignore the suggestion if so.

File: content/en/tracing/trace_collection/library_config/php.md [hunk 4]
```diff
@@ -313,6 +313,19 @@ Set an application's version in traces and logs, for example: `1.2.3`, `6c44da20
...
 A comma-separated list of query parameters to be collected as part of the URL. Set to empty to prevent collecting any parameters, or `*` to collect all parameters. Added in version `0.74.0`.
 
+`DD_TRACE_HTTP_POST_DATA_PARAM_ALLOWED`
+: **INI**: `datadog.trace.http_post_data_param_allowed`<br>
+**Default**: ""<br>
+A comma-separated list of HTTP POST data fields to be collected as part the request. Leave empty to redact all posted values, or use `*` to collect and redact them if needed. When specific fields are explicitly allowed, only those fields' values remain. Added in version `0.86.0`.<br>
+**Example**: 
+  - The posted data is `foo[baz]=bar&foo[baz][bar]=quz&foo[bar][password]=baz&foo[bar][baz]=quz`
+  - `DD_TRACE_HTTP_POST_DATA_PARAM_ALLOWED` is set to `foo.baz,foo.bar.password`<br>
+  - In this scenario, the collected metadata is:
```
Comment (pierotibou) [inline content/en/tracing/trace_collection/library_config/php.md:323]: Are you sure of the example @PROFeNoM ? I would have assumed something different

Comment (PROFeNoM) [reply content/en/tracing/trace_collection/library_config/php.md:323]: Yes, there was indeed something wrong

File: content/en/tracing/trace_collection/library_config/php.md [hunk 5]
```diff
@@ -313,6 +313,19 @@ Set an application's version in traces and logs, for example: `1.2.3`, `6c44da20
 **Default**: `*`<br>
 A comma-separated list of query parameters to be collected as part of the URL. Set to empty to prevent collecting any parameters, or `*` to collect all parameters. Added in version `0.74.0`.
 
+`DD_TRACE_HTTP_POST_DATA_PARAM_ALLOWED`
+: **INI**: `datadog.trace.http_post_data_param_allowed`<br>
+**Default**: ""<br>
+A comma-separated list of HTTP POST data fields to be collected. Leave empty if you don't want to collect any posted values. When setting this value to the wildcard `*`, all posted data will be collected, but the values for fields that match the `DD_TRACE_OBFUSCATION_QUERY_STRING_REGEXP` obfuscation rule will be redacted. If specific fields are given, then only these fields' values are collected in clear, while the values for all other fields will be redacted. Added in version `0.86.0`.<br>
```
Comment (buraizu) [inline content/en/tracing/trace_collection/library_config/php.md:319]:
```suggestion
A comma-separated list of HTTP POST data fields to be collected. Leave empty if you don't want to collect any posted values. When setting this value to the wildcard `*`, all posted data is collected, but the values for fields that match the `DD_TRACE_OBFUSCATION_QUERY_STRING_REGEXP` obfuscation rule are redacted. If specific fields are given, then only these fields' values are visible, while the values for all other fields are redacted. Added in version `0.86.0`.<br>
```

**Review by buraizu — APPROVED** [review]
Thanks for clarifying that behavior. I'm approving with a few modifications to the new text.

**Comment by PROFeNoM** [comment]
Hey @buraizu ! 

Any updates on when this could be merged? :)

**Pull request merged**